package.json

```diff
@@ -13,10 +13,9 @@
         "lint:workflows": "find .github/workflows -type f \\( -iname '*.yaml' -o -iname '*.yml' \\) | xargs -I {} sh -c 'echo \"Linting {}\"; action-validator \"{}\"'",
         "lint:prettier": "prettier --check .",
         "lint:prettier:fix": "prettier --write .",
-        "lint:js": "eslint --max-warnings 0 modules/*/element-web/src packages/*/src",
-        "lint:ts": "for W in @element-hq/element-web-module-api nordeck-plugin; do yarn workspace $W run lint; done",
-        "lint:py": "yarn workspace @element-hq/synapse-guest-module run lint",
-        "lint": "yarn workspaces run lint",
+        "lint:eslint": "eslint --max-warnings 0 modules/*/element-web/src packages/*/src",
+        "lint:types": "yarn workspaces run lint:types",
+        "lint:codestyle": "yarn workspaces run lint:codestyle",
```

Comment:
The primary goal of this PR is to be able to avoid needing to list individual submodules in these commands